components/form-fields/dynamic-checkbox-group.jsx

```diff
@@ -75,7 +75,6 @@ DynamicCheckboxGroup.propTypes = {
 };
 
 DynamicCheckboxGroup.defaultProps = {
-  colNum: 1,
```

Comment:
Why’d this get removed? The prop is still referenced in the file.

Reply:
Thanks for pointing this out—I'm going to remove the rest of the prop references.